src/main/java/pl/pszczolkowski/mustdo/web/restapi/tasksList/TasksListNew.java

```diff
@@ -0,0 +1,41 @@
+package pl.pszczolkowski.mustdo.web.restapi.tasksList;
+
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel
+public class TasksListNew {
+
+   @NotNull
+   @Size(min = 3,
+      max = 100)
```

Comment:
You are so annoing with this "Formatting" that i changed that.... Btw i was using the project style formatting....